src/System.Net.Primitives/tests/PalTests/SocketAddressPalTest.cs

```diff
@@ -188,5 +188,15 @@ public void IPv6AddressAndPort_Get_Set(byte[] address, uint scope, ushort port)
             Assert.Equal(scope, actualScope);
             Assert.Equal(port, SocketAddressPal.GetPort(buffer));
         }
+
+        // This is platform specific test. So far Windows, Linux and OSX use same layout.
+        [Fact]
+        public void SocketAddress_To_String()
```

Comment:
nit: This test method name doesn't follow our convention. The first part of the name is the property/method being tested, etc.  I would suggest this be "ToString_ValidSocketAddress_ExpectedFormat()